msal/src/main/java/com/microsoft/identity/client/internal/CommandParametersAdapter.java

```diff
@@ -357,6 +364,220 @@ public static DeviceCodeFlowCommandParameters createDeviceCodeFlowCommandParamet
         return commandParameters;
     }
 
+    /**
+     * Creates command parameter for [SignUpStartCommand] of Native Auth.
```

Comment:
You can use `{@link SignUpStartCommand}` to actually link to that file from within the javadoc....here and everywhere

Reply:
Updated